tensilelite/Tensile/KernelWriter.py

```diff
@@ -3943,18 +3943,18 @@ def readWriteVectors(mat, vw, kernel):
     self.states.b.numVgprLocalWriteAddr = 0 if kernel["LocalWriteUseSgprB"] else 1 * self.states.rpla
 
     if self.states.archCaps["HasLDSGT64K"] and not kernel["1LDSBuffer"] and not kernel["LocalWriteUseSgprA"] :
```

Comment:
```suggestion
    if self.states.archCaps["HasLDSGT64K"] and not kernel["LocalWriteUseSgprA"] :
```
We still need more VGPRs when LdsNumElementsAligned is large, even if 1LDSBuffer is enabled.